src/components/vuestic-components/va-form/VaForm.vue

```diff
@@ -0,0 +1,162 @@
+<script>
+import { ContextPluginMixin, getContextPropValue } from '../../context-test/context-provide/ContextPlugin'
+
+const availableFormTags = ['VaInput', 'VaColorInput', 'VaDatePicker', 'VaCheckbox', 'VaRadio', 'VaSelect', 'VaFileUpload']
+const getAllChildren = (vm, children = []) => {
+  vm.$children.forEach(function (child) {
+    children.push(child)
+    child.$children.length > 0 && getAllChildren(child, children)
+  })
+  return children
+}
+
+export default {
+  name: 'VaForm',
+  mixins: [ContextPluginMixin],
+  props: {
+    lazyValidation: {
+      type: Boolean,
+      default () {
+        return getContextPropValue(this, 'lazyValidation', false)
+      },
+    },
+    autofocus: {
+      type: Boolean,
+      default () {
+        return getContextPropValue(this, 'autofocus', false)
+      },
+    },
+    tag: {
+      type: String,
+      default () {
+        return getContextPropValue(this, 'tag', 'div')
+      },
+    },
+  },
+  data () {
+    return {
+      valid: true,
```

Comment:
I'm not seeing any purpose, keeping valid state for form itself.

Reply:
This need to implement focusInvalid method at the form and resetValidation

Reply:
@Kreezag 

Still not seeing any use. Why not make these dynamic?

Reply:
Yes, I can remove this field